engine/language_client_codegen/src/python/mod.rs

```diff
@@ -105,6 +106,8 @@ impl TryFrom<(&'_ IntermediateRepr, &'_ crate::GeneratorArgs)> for PythonInit {
     fn try_from((_, gen): (&'_ IntermediateRepr, &'_ crate::GeneratorArgs)) -> Result<Self> {
         Ok(PythonInit {
             default_client_mode: gen.default_client_mode.clone(),
+            // TODO: Should we use gen.version instead?
```

Comment:
Seems better to me to grab the version from generators file, because that's what the error message implies.

I think it might be impossible for the generators file to be out of sync with CARGO_PKG_VERSION by the time codegen happens, meaning that there's no difference between (a) grabbing the version from CARGO_PKG_VERSION or (b) taking it from generators block. But it may be better not to rely on that fact. For example, discrepancies would exist between the two if the user is on a higher patch-version of the playground.

To say it another way - it would be confusing if the error messages said "your generators files says '0.1.0'" if their generators file actually says '0.1.1'.

Reply:
resolved with better names!